README.rst

```diff
@@ -97,10 +97,14 @@ Setup (integrate in an existing django project)
         'drf_yasg',
     ]
 
-also add ``AUTH_USER_MODEL`` and ``SITE_ID`` to your ``settings.py``::
+also add ``AUTH_USER_MODEL``, ``SITE_ID`` and ``AUTHENTICATION_BACKENDS``
+to your ``settings.py``::
 
     AUTH_USER_MODEL = 'openwisp_users.User'
     SITE_ID = 1
+    AUTHENTICATION_BACKENDS = [
+        'openwisp_users.backends.UsersAuthenticationBackend',
+    ]
```

Comment:
@NoumbissiValere please don't forget to add information like this one to the README: every time we add something to the test env we have to add it to the setup instructions too.